src/implementation/Client/HTTPClient/configuration.ts

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import IClientConfiguration from '../../../interfaces/Client/IClientConfiguration';
+import { KeyValueType } from '../../../types/KeyValue.type';
+import { GetConfigurationResponse as GetConfigurationResponseResult } from '../../../types/configuration/GetConfigurationResponse';
+import HTTPClient from './HTTPClient';
+import { SubscribeConfigurationCallback } from '../../../types/configuration/SubscribeConfigurationCallback';
+
+// https://docs.dapr.io/reference/api/configuration_api
```

Comment:
Link does not exist. Can we use a working link? Like https://docs.dapr.io/developing-applications/building-blocks/configuration/howto-manage-configuration/ - it clearly calls out that HTTP is currently unsupported.

Similarly, the Error message for the functions below can also mention that HTTP is currently unsupported, instead of just saying that it's not implemented.

Reply:
As a nit, the error message string can go to a constant above.

Reply:
Agreed, changed it